peripherals/syscfg/syscfg_f303_3x8.yaml

```diff
@@ -0,0 +1,276 @@
+      PD13: [3, "Select PD13 as the source input for the EXTI13 external interrupt"]
+      PE13: [4, "Select PE13 as the source input for the EXTI13 external interrupt"]
+      PF13: [5, "Select PF13 as the source input for the EXTI13 external interrupt"]
+      PG13: [6, "Select PG13 as the source input for the EXTI13 external interrupt"]
+    EXTI14:
+      PA14: [0, "Select PA14 as the source input for the EXTI14 external interrupt"]
+      PB14: [1, "Select PB14 as the source input for the EXTI14 external interrupt"]
+      PC14: [2, "Select PC14 as the source input for the EXTI14 external interrupt"]
+      PD14: [3, "Select PD14 as the source input for the EXTI14 external interrupt"]
+      PE14: [4, "Select PE14 as the source input for the EXTI14 external interrupt"]
+      PF14: [5, "Select PF14 as the source input for the EXTI14 external interrupt"]
+      PG14: [6, "Select PG14 as the source input for the EXTI14 external interrupt"]
+    EXTI15:
+      PA15: [0, "Select PA15 as the source input for the EXTI15 external interrupt"]
+      PB15: [1, "Select PB15 as the source input for the EXTI15 external interrupt"]
+      PC15: [2, "Select PC15 as the source input for the EXTI15 external interrupt"]
+      PD15: [3, "Select PD15 as the source input for the EXTI15 external interrupt"]
+      PE15: [4, "Select PE15 as the source input for the EXTI15 external interrupt"]
+      PF15: [5, "Select PF15 as the source input for the EXTI15 external interrupt"]
+      PG15: [6, "Select PG15 as the source input for the EXTI15 external interrupt"]
+  CFGR2:
+    SRAM_PEF:
+      _read:
+        NoParityError: [0, "No SRAM parity error detected"]
+        ParityErrorDetected: [1, "SRAM parity error detected"]
+      _write:
+        Clear: [1, "Clear SRAM parity error flag"]
+    BYP_ADDR_PAR:
+      NoBypass: [0, "The ramload operation is performed taking into consideration bit 29 of the address when the parity is calculated"]
+      Bypass: [1, "The ramload operation is performed without taking into consideration bit 29 of the address when the parity is calculated"]
+    SRAM_PARITY_LOCK:
+      Disconnected: [0, "SRAM parity error disconnected from TIM1/15/16/17 Break input"]
+      Connected: [1, "SRAM parity error connected to TIM1/15/16/17 Break input"]
+    LOCKUP_LOCK:
+      Disconnected: [0, "Cortex-M4F LOCKUP output disconnected from TIM1/15/16/17 Break input"]
+      Connected: [1, "Cortex-M4F LOCKUP output connected to TIM1/15/16/17 Break input"]
+  # TODO: Double effort for this and 3x4
+  CFGR3:
+    ADC2_DMA_RMP:
+      # TODO: Naming
+      MapDma2: [0, "ADC2 mapped on DMA2"]
+      MapDma1Ch2: [3, "ADC2 mapped on DMA1 channel 2"]
+      MapDma1Ch4: [4, "DC2 mapped on DMA1 channel 4"]
+    I2C1_TX_DMA_RMP:
+      MapDma1Ch6: [0, "I2C1_TX mapped on DMA1 CH6"]
+      MapDma1Ch2: [1, "I2C1_TX mapped on DMA1 CH2"]
+      MapDma1Ch4: [2, "I2C1_TX mapped on DMA1 CH4"]
+      # TODO: Name doubled value??
```

Comment:
We can't have duplicate names, so depending on the situation we've either omitted the duplicate variant or named it something to distinguish it. For this I'd probably just omit it unless there's any reason to use one over the other?

Reply:
There is no reason to add the duplicated, so I do as you suggested.